tests/openassetio/test_logging.py

```diff
@@ -97,29 +99,34 @@ def test_constructor(self, mock_logger, monkeypatch):
                 (lg.LoggerInterface.kError, lg.LoggerInterface.kError),
                 (lg.LoggerInterface.kCritical, lg.LoggerInterface.kCritical)
         ):
-            monkeypatch.setenv(var, str(value))
+            monkeypatch.setenv(severity_control_envvar, str(value))
             f = lg.SeverityFilter(mock_logger)
             assert f.getSeverity() == expected
 
-    def test_severity_accessors(self, severity_filter):
-        for severity in self.all_severities:
+
+class Test_SeverityFilter_methods():
```

Comment:
[consider] This could be two separate tests classes for `Test_SeverityFilter_setSeverity` and `Test_SeverityFilter_log`.

Reply:
Split [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/214/commits/2e3b34fd17b53f1924a20b5ab4cb852566a13c48).